### PR TITLE
don't build tags in Travis

### PR DESCRIPTION
**Pull request by mossprescott**
A quick change to conserve Travis’s meager resources by _not_ building from the tags; the tags are always applied to a commit which has _already_ been built and uploaded to github releases.

I’m not 100% confident that this syntax is correct, but I know of no way to test it other than merging. See https://docs.travis-ci.com/user/customizing-the-build/#Using-regular-expressions.

@sellout: I don't know if you’re already woking on a more significant change to the build scripts, but this seems like a good thing to get in ASAP.

### CODE REVIEW

**Comment by sellout** [comment]
Yeah, mine is basically this plus disabling the REPL tag (if @damon says that's ok).

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ notifications:
   slack:
     secure: KPCXtCAjtlpRQcH7TQtprrf67yIqPVpht9L6/+wbbNd14Tj3x53fiZ3OTabHVx3SL2ZWxbWq+FJGxdIFMyxC/fC0bV9flB7PlRtqHJV1Q7g49odAMAZL7syWr7vFP5bJt+TXuxHvMBihDHFk1Cehy4a1cnrOMReDgqGIFCwQtbI=
 
+branches:
+  except:
+    - /^v\d+\.\d+\.\d+-.*$/ # don't redundantly build tags
```
Comment (sellout) [inline .travis.yml]: `.*$`?!

Comment (mossprescott) [inline .travis.yml]: Yeah, it's dumb but it's what they did in their example, so I just copied it out fear that Ruby regexes might be weirder than I imagine. Presumably neither is actually needed.

**Comment by mossprescott** [comment]
@sellout So, merge this or wait for a verdict on the REPL?

**Comment by sellout** [comment]
Oh, definitely merge it.

**Comment by drostron** [comment]
👍🏾

**Comment by drostron** [comment]
Needs a merge from master

**Comment by mossprescott** [comment]
Oops. That one was my fault. Building again.

**Pull request merged**